pipeline/process/base/reconciler.py

```diff
@@ -158,6 +158,7 @@ def reconcile(self, rec, reconcileType="all"):
         if reconcileType in ['all', 'name']:
             # Get name from Record
             vals = self.extract_names(rec)
+            vals = vals.sort(key=len,reverse=True)
```

Comment:
Should be either `vals.sort()` or `vals = sorted(vals, ...)` Current code will assign None to vals